pagerduty/resource_pagerduty_incident_workflow.go

```diff
@@ -14,6 +15,8 @@ import (
 	"github.com/heimweh/go-pagerduty/pagerduty"
 )
 
+const inlineStepSchemaLevel = 1
```

Comment:
Controls how many levels of inline steps we allow. Currently only 1 level is allowed, ie an inline step may not use an action that itself uses an inline step.